### PR TITLE
fix tunnel issue

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @churkin @miherlosev

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,33 +61,40 @@ Example
...
-    	}], function(e) {
-    		console.log('All workers are created!')
-    	});
-    });
+```js
+var testCafeBrowserStack = require('testcafe-browserstack');
+
+testCafeBrowserStack.init({
+    TestCafe: {
+        hostname: 'lavrov-w8',
```
Comment (miherlosev) [inline README.md]: Revert back to `'127.0.0.1'`

**Comment by LavrovArtem** [comment]
FPR

File: README.md [hunk 2]
```diff
@@ -61,33 +61,40 @@ Example
...
+        accessKey: 'accessKey',
+        localTesting: true
+    }
+}, function(err) {
+    if (!err) {
+        testCafeBrowserStack.createWorkers([{
+            os: 'OS X',
+            os_version: 'Snow Leopard',
+            browser: 'chrome',
+            browser_version: '40.0',
```
Comment (miherlosev) [inline README.md]: Let's drop browser version to run on the latest chrome version.

Comment (LavrovArtem) [reply README.md]: `Error: Validation Failed - `browser_version` invalid`

**Comment by LavrovArtem** [comment]
FPR

**Comment by LavrovArtem** [comment]
ping

**Pull request merged**